### PR TITLE
backendBuild-PullRequest

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
**About this project**<br/>

This is an app that displays random greetings. This is the backend  part.

**For it:**<br/>
- The backend-build featured branch is created.
- A new Rails API is created.
- The project is Initialized  with Git.
- The project has a Postgres database set up.
- A table for storing your messages and create 5 different greetings has been created.
- A controller with action, and a rails route has been built to create an API endpoint that selects a random greeting from the table.
- Linters were passed.

**Goal:**<br/>
The goal is to understand and feel comfortable with Rails and connecting it to React.

[FRONTEND](https://github.com/IvonneBenitesRodriguez/hello-react-frontend/pull/1)

### CODE REVIEW

File: app/controllers/greetings_controller.rb
Comment (shahadat3669) [inline app/controllers/greetings_controller.rb:1]: 👍 Excellent progress on your project! Just one more suggestion:

- [x] **Remove Redundant Files:** It appears that there are two `greetings_controller.rb` files in your codebase, and  this one is not being used. To maintain a clean and organized project structure, it's recommended to remove any redundant or unused files. Please delete the unnecessary `greetings_controller.rb` file from your project.

This practice will help streamline your codebase and avoid confusion for anyone reviewing or collaborating on the project. Keep up the great work! 🚀🗑️

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @IvonneBenitesRodriguez , 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ the API endpoint for returning a random greeting was created
✅ a database table for storing different greetings was created
✅ the two apps work together

## Required Changes ♻️

- [ ] **Please check also the frontend PR for changes.**

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadat3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/greetings_controller.rb
Comment (IvonneBenitesRodriguez) [reply app/controllers/greetings_controller.rb:1]: Dear @shahadat3669 👋🏾
Thank you so much for your kind suggestions and words,🌷
I appreciate it. I modified the change you suggested,
Have a beautiful week and thank you again, ⭐️
Best regards,
Ivonne

**Review by shahadat3669 — APPROVED** [review]
Hi @IvonneBenitesRodriguez,

Your project is complete!  There is nothing else to say other than it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ the API endpoint for returning a random greeting was created
✅ the two apps work together
✅ The code is top-notch! 💯

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table  width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadat3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**